aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs

```diff
@@ -221,9 +221,15 @@ impl TransactionGasParameters {
         }
     }
 
-    pub fn storage_fee_for_bytes(&self, key: &StateKey, op: &WriteOp) -> Fee {
+    pub fn storage_fee_for_bytes(
```

Comment:
this is confusing - why are we passing op and computing it's bytes, when maybe_group_size is passed?
If bytes are None (when is that the case), is correct charge 0 or from maybe_group_size?


Maybe we should have an enum:

```
trait StorageChargeableItem {
  Bytes(WriteOp),
  Explicit(u64)
}

```

maybe you also need Deletion, if we really need 0  when maybe_group_size is set

Reply:
so value size is 0 both for deletion and empty bytes, right?

Reply:
Good point, seems we need to take value_size: Option<usize> to be able to do the same thing as before